# Definitions of traceable obstacles
# Identifier corresponds to Traceable ID set in MoCap

node_rate: 100
delta_t: 0.05
horizon_N: 30
obstacle:
      pose_measured: /Target2/pose
      state_estimation: /Target2/obstacleStateEstimation
      state_prediction: /Target2/obstacleStatePrediction
